test/correctness/image_io.cpp

```diff
@@ -1,8 +1,10 @@
 #include <fstream>
 
+#include "test/common/halide_test_dirs.h"
```

Comment:
This will not pass our lint rules; #include files should be in alphabetical order; this implies that they must not be order-dependent.

Reply:
Yep, it was ugly. Now it’s a little better.